neovim_gui/cli.py

```diff
@@ -1,26 +1,117 @@
 """CLI for accessing the gtk/tickit UIs implemented by this package."""
+import os
+import resource
+import sys
 import shlex
 
 import click
+import yaml
 
 from .ui_bridge import UIBridge
 from neovim import attach
 from neovim.compat import IS_PYTHON3
 
 
+CONFIG_FILES = (
+    '.pynvim.yaml',
+    '~/.pynvim.yaml',
+    '~/.config/pynvim/config.yaml'
+)
+
+
+def load_config(config_file):
+    """Load config values from yaml."""
+
+    if config_file:
+        with open(config_file) as f:
+            return yaml.load(f)
+
+    else:
+        for config_file in CONFIG_FILES:
+            try:
+                with open(os.path.expanduser(config_file)) as f:
+                    return yaml.load(f)
+
+            except IOError:
+                pass
+
+    return {}
+
+
+# http://code.activestate.com/recipes/278731-creating-a-daemon-the-python-way/
+def detach_proc(workdir='.', umask=0):
+    """Detach a process from the controlling terminal and run it in the
+    background as a daemon.
+    """
+
+    # Default maximum for the number of available file descriptors.
+    MAXFD = 1024
+
+    # The standard I/O file descriptors are redirected to /dev/null by default.
+    if (hasattr(os, "devnull")):
+        REDIRECT_TO = os.devnull
+    else:
+        REDIRECT_TO = "/dev/null"
+
+    try:
+        pid = os.fork()
+    except OSError, e:
```

Comment:
After digging around for best practices how to fork (seems a science) I found this script at http://code.activestate.com/recipes/278731-creating-a-daemon-the-python-way/

I could live without reraising another general Exception.